label_studio/data_export/admin.py

```diff
@@ -1,5 +1,4 @@
 """This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
 """
-from django.contrib import admin
```

Comment:
perhaps we should just delete this file instead?

Reply:
yes from what I can tell its safe to delete.